packages/design-system-docs/src/pages/startup/sass-and-css.md

```diff
@@ -3,38 +3,95 @@ title: Using Sass/CSS
 
-1. Download the zip file from the latest design system release and open that file.
 1. Copy the `dist/css/index.css` file into a relevant place in your code base — likely a directory where you keep third-party libraries. In the example below, our directory is `css/vendor`.
-1. Add a `<link>` to the stylesheet in your site's `<head>`
+2. Add a `<link>` to the stylesheet in your site's `<head>`
 
 For example:
 
 ```html
 <link rel="stylesheet" src="/css/vendor/design-system/index.css" />
 ```
 
-[View an example](https://github.com/CMSgov/design-system/blob/master/examples/article/index.html)
+3. [Ensure fonts and images are rendered correctly.]({{root}}/startup/sass-and-css/#using-fonts-and-images)
 
-<h4 class="ds-h4 ds-u-font-size--base" id="sass">Sass</h4>
+## Using fonts and images
 
-If you're already using Sass to style your site, another way to include the design system's styles is by importing its un-minified Sass file from NPM.
+By default, the design system expects a file structure like this:
 
-1. First, make sure your build system is configured so that the `node_modules` directory is in the list of Sass [`includePaths`](https://github.com/sass/node-sass#includepaths).
-2. Add the following to your Sass file:
+```
+└── Your site's build folder/
+    ├── images/
+    ├── fonts/
+    └── index.css
+```
+
+You can manually copy the image and font directories, or you could integrate this step into your build process.
+
+- [Here's an example of copying assets using Gulp](https://github.com/CMSgov/design-system/blob/master/examples/react-app/Gulpfile.js).
+- [Here's an example of copy assets using Webpack](https://github.com/CMSgov/design-system/blob/master/examples/webpack-demo/webpack.config.js).
+
+Alternatively if your project uses Webpack or Create React App, you can [configure the bundler to process the assets]({{root}}/startup/sass-and-css/#setting-asset-paths-to-node_modules).
+
+### Overriding asset paths
+
+If you are using Sass/SCSS, you can change the default asset paths by overriding the `$font-path` and `$image-path` Sass variables.
+
+For example, if your project build directory file structure looked like this:
+
+```
+└── Your site's build folder/
+    ├── images/
+    ├── fonts/
+    └── css/
+      └── index.css/
+```
+
+The asset paths would have to be redefined like so:
 
 ```css
-@import '{{npm}}/dist/scss/index';
+$font-path: '../fonts';
+$image-path: '../image';
 ```
 
-[Learn how to override and theme Sass variables]({{root}}/startup/theming/).
+Without overriding these variables, it's possible that your fonts and images will not render correctly in your project.
+
+#### Setting asset paths to `node_modules`
+
+If your project uses Webpack or Create React App, you can set the asset paths directly to the fonts and image folders in your `node_modules`.
+
+```css
+$font-path: '~{{npm}}/dist/fonts';
+$image-path: '~{{npm}}/dist/images';
+```
```

Comment:
Which file do is this added to?

Reply:
Do you mean where is the font and image paths defined? It just has to happen in a CSS file before you import the design system. I recommend looking through the child design system example or any of the hc.gov or m.gov child design systems

Reply:
https://github.com/CMSgov/design-system/pull/763/files#diff-c637d2d90071b64e35d7490418de01f2R86